spec.html

```diff
@@ -1508,28 +1510,28 @@ <h1>Number::exponentiate ( _base_, _exponent_ )</h1>
           <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-dependent approximation of the result of raising _base_ to the power _exponent_, subject to the following requirements:</p>
           <ul>
             <li>If _exponent_ is *NaN*, the result is *NaN*.</li>
-            <li>If _exponent_ is *+0*, the result is 1, even if _base_ is *NaN*.</li>
-            <li>If _exponent_ is *-0*, the result is 1, even if _base_ is *NaN*.</li>
+            <li>If _exponent_ is *+0*, the result is *1*<sub>𝔽</sub>, even if _base_ is *NaN*.</li>
+            <li>If _exponent_ is *-0*, the result is *1*<sub>𝔽</sub>, even if _base_ is *NaN*.</li>
```

Comment:
Side note: "even if ..." is so weird. Is that clarification really necessary?

Reply:
Thank you very much for reviews so far, they are really valuable!

> @caiolima
> 
> > Removing subscript of operators and removing all comparisons using '+0' as operand
> 
> Why remove comparisons using +0? I think some of these were best left alone. Like I see cases where we changed it say the mathematical value of an integral Number is ≥ 0 instead of saying the integral Number is ≥ **+0**. Those statements are equivalent (IEEE defines this inequality), and the new one is doing a conversion for no real reason.
> 
> I'm assuming you were just trying to stay in the mathematical value space as much as possible (and that's a good thing), but I don't think we need to do that for simple comparisons.

You guessed it. The main idea was to only have inequalities using mathematical values. Form previous comments, I understood that this would be a thing we would follow, but seems like that was a misunderstanding of mine. I personally like this idea of having inequalities only defined to MV, but I'm also not against allowing it to other numeric types.